# when i use random for the first time my file name was random.py
# which results name conflict and take me   a long time to find out the error!!
import random
random_int = random.randint(1,100)
print(f"i am just generating random integer:- {random_int}")
